content/en/observability_pipelines/production_deployment_overview/_index.md

```diff
@@ -1,42 +1,188 @@
+Choose the source that aligns with your protocol. Each Observability Pipelines Worker source implements different protocols. For example, Observability Pipelines Worker sources and sinks use gRPC for inter-Observability Pipelines Worker communication, and the HTTP source allows you to receive data over HTTP. See [Sources][2] for their respective protocols.
+
+## Collecting data
+
+Your pipeline begins with data collection. Your services and systems generate logs, metrics, and traces that can be collected and sent downstream to your destinations. Data collection is achieved with agents, and understanding which agents to use ensures you are collecting the data you want.
+
+### Choosing agents
+
+You should choose the agent that optimizes your engineering team's ability to monitor their systems. Therefore, integrate Observability Pipelines Worker with the best agent for the job and replace the other agents with the Observability Pipelines Worker. 
+
+#### When Observability Pipelines Worker can replace agents
+
+The Observability Pipelines Worker can replace agents performing generic data forwarding functions, such as:
+
+- Tailing and forwarding log files
+- Collecting and forwarding service metrics without enrichment
+- Collecting and forwarding service logs without enrichment
+- Collecting and forwarding service traces without enrichment
+
+These functions collect and forward existing data without modifying data. Since these functions are not unique, these agents can be replaced with the Observability Pipelines Worker to provide more configuration options that may be needed as your environment evolves. 
+
+If you decide to replace an agent, configure Observability Pipelines Worker to perform the same function as the agent you are replacing. Use source components such as the `file`, `journald`, and `host_metrics` sources to collect and forward data. You can process data locally on the node or remotely on your aggregators. See [Choosing where to process data](#choosing-where-to-process-data) for more information.
+
+{{< img src="observability_pipelines/production_deployment_overview/as_an_agent.png" alt="A diagram showing a node containing multiple services and the Observability Pipelines Worker, where the services are sending data to the Worker and the Worker is sending data out" style="width:30%;" >}}
+
+#### When Observability Pipelines Worker should integrate with agents
+
+The Observability Pipelines Worker should integrate with agents that produce vendor-specific data that the Observability Pipelines Worker cannot replicate.
+
+For example, Datadog [Network Performance Monitoring][4] integrates the Datadog Agent with vendor-specific systems and produces vendor-specific data. Therefore, the Datadog Agent should collect the data and send it directly to Datadog, since the data is not a supported data type in the Observability Pipelines Worker.
```

Comment:
This paragraph should follow line 54 ("You should choose the agent that optimizes your engineering team's ability to monitor their systems....")